awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -244,10 +244,10 @@ function JobTemplateForm({
             value={inventory}
             onBlur={() => inventoryHelpers.setTouched()}
             onChange={value => {
-              inventoryHelpers.setValue(value.id);
+              inventoryHelpers.setValue(value ? value.id : null);
```

Comment:
Not necessarily within the scope of this PR but I do think there is some value in setting the value of the Inventory (formik value) to the whole inventory object and then parsing through that object on submit.  It would would eliminate a 2nd state handler like we have on line 248.

Reply:
Yeah, I think I agree, though this fix would be bigger than the rest of this PR, so I'll defer it for later.